kview/src/main/java/dev/ikm/komet/kview/mvvm/view/journal/JournalController.java

```diff
@@ -1145,8 +1150,20 @@ private void loadNavigationPanel(PublicIdStringKey<ActivityStream> navigationAct
             }
         });
 
-        navigatorNodePanel = (Pane) navigatorNode.getNode();
-        setupSlideOutTrayPane(navigatorNodePanel, navSlideoutTrayPane);
+        return (Pane) navigatorNode.getNode();
+    }
+
+    private void loadPatternNavigationPanel(PublicIdStringKey<ActivityStream> navigationActivityStreamKey,
```

Comment:
Thanks for taking up this story. Can we change the name of this method to just loadNavigationPanels since it loads both Concept and Pattern navigators?
   The existing loadNavigationPanel can be changed to loadClassicConceptNavigator.

Reply:
good idea... will do